docroot/modules/custom/layout_builder_custom/layout_builder_custom.module

```diff
@@ -794,6 +797,10 @@ function _layout_builder_custom_process_element(array $element, FormStateInterfa
 
       break;
 
+    case 'inline_block:uiowa_button':
+      $element["field_uiowa_button_link"]["widget"][0]["title"]["#maxlength"] = 25;
```

Comment:
Please change to single quotes :)

Reply:
can do! I'm sorry lol, when i cope th path from PHPStorm it does doubles!